### PR TITLE
use ugFont for standard input styles

### PR DESCRIPTION
**Pull request by BrianHicks**

### CODE REVIEW

**Comment by BrianHicks** [comment]
@NoRedInk/design does the Percy diff here look OK to you?

**Comment by staceyadams** [comment]
LGTM 👍

**Comment by tesk9** [comment]
What is happening here? We're changing all of our input styles from basefont to ugfont?

**Comment by BrianHicks** [comment]
so here's the deal… I'm not sure this is the right thing to do. I thought it'd be fine but the Percy diff is making me think it's not. I requested reviews before it was done; sorry for wasting y'all's time!

The point of what I'm doing is to get the ugFont for the writing prompt and passage editor. Buuuut I don't want to have an override on those specific editors. @tesk9 would it match your intent in designing this module to add a specific `UserGenerated` case?

**Comment by tesk9** [comment]
I don't think I made InputStyles -- I don't have strong opinions on how these styles ought to work. Having a separate UserGenerated style sounds fine to me

**Comment by BrianHicks** [comment]
ok! now it's separate and the other text inputs won't change.

File: script/deprecated-imports.csv [hunk 1]
```diff
@@ -8,8 +8,10 @@ src/Nri/Ui/Page/V3.elm,Nri.Ui.Text,2
 src/Nri/Ui/SlideModal/V2.elm,Nri.Ui.Button,8
 src/Nri/Ui/SlideModal/V2.elm,Nri.Ui.Icon,3
 src/Nri/Ui/SlideModal/V2.elm,Nri.Ui.Text,2
+src/Nri/Ui/TextInput/V6.elm,Nri.Ui.InputStyles,2
```
Comment (tesk9) [inline script/deprecated-imports.csv:11]: Should we update these..?

Comment (BrianHicks) [reply script/deprecated-imports.csv:11]: I mean, yeahhhhhhhhhhhh but also



If you really think we should, we can, but it seems to require bumping both versions to their next `Vwhatever` and this is already going kind of off our critical path for TDA right now.

Comment (tesk9) [reply script/deprecated-imports.csv:11]: Oh -- i'm surprised that's not a patch change! happy to wait then

**Pull request merged**